services/src/quic_connection_utils.rs

```diff
@@ -79,6 +79,18 @@ impl QuicConnectionUtils {
         let timeout = IdleTimeout::try_from(Duration::from_secs(1)).unwrap();
         transport_config.max_idle_timeout(Some(timeout));
         transport_config.keep_alive_interval(Some(Duration::from_millis(500)));
+
+        const DATAGRAM_RECEIVE_BUFFER_SIZE: usize = 1024 * 1024 * 1024;
+        const DATAGRAM_SEND_BUFFER_SIZE: usize = 1024 * 1024 * 1024;
+        const INITIAL_MAXIMUM_TRANSMISSION_UNIT: u16 = MINIMUM_MAXIMUM_TRANSMISSION_UNIT;
+        const MINIMUM_MAXIMUM_TRANSMISSION_UNIT: u16 = 1280;
+        transport_config.datagram_receive_buffer_size(Some(DATAGRAM_RECEIVE_BUFFER_SIZE));
+        transport_config.datagram_send_buffer_size(DATAGRAM_SEND_BUFFER_SIZE);
+        transport_config.initial_mtu(INITIAL_MAXIMUM_TRANSMISSION_UNIT);
```

Comment:
not sure if setting the initial value will prevent that heuristic will move the packet size up.
**would expect to have a max (or flex) size**

Reply:
this?
![image](https://github.com/blockworks-foundation/lite-rpc/assets/95291500/31b97314-30e4-4bf4-81e9-5a35c65741c6)